sdk/storage/azure-storage-blob/tests/test_blob_tags_async.py

```diff
@@ -339,13 +402,17 @@ async def test_list_blobs_returns_tags(self, storage_account_name, storage_accou
 
         #Assert
         async for blob in blob_list:
-            self.assertEqual(blob.tag_count, len(tags))
+            assert blob.tag_count == len(tags)
             for key, value in blob.tags.items():
-                self.assertEqual(tags[key], value)
+                assert tags[key] == value
 
     @pytest.mark.playback_test_only
```

Comment:
This probably doesn't need to be playback only? It's not in sync :)

Reply:
Good catch!